### PR TITLE
Automatic fileds population

### PR DESCRIPTION
**Pull request by Bryksin**
When creating new Employee and liking to the existed user - this fix will populate not only email address field, but also full name and avatar image.

That should save a lot of time and avoid typos - especially when you start from scratch and have to convert 30+ users into employees

### CODE REVIEW

**Comment by jholze** [comment]
:+1:

**Comment by Yenthe666** [comment]
Oooh that is a nice and handy extra! :+1: 
Nice one.

**Comment by Briksins** [comment]
We just faced that problem that have to convert all employees from our LDAP into Employees, and typing names for each one is nightmare, especially in multicultural environment, too many typos in names.
Also picture was not even possible to set, but now it coped automatically :)

Glad you liked this change :+1:

**Comment by Yenthe666** [comment]
@antonylesuisse and @odony could you guys have a look at this? This would be a great addition for Odoo 9! :smile:

File: addons/bus/bus.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def loop(self):
         with openerp.sql_db.db_connect('postgres').cursor() as cr:
             conn = cr._cnx
             cr.execute("listen imbus")
-            cr.commit();
+            cr.commit()
```
Comment (odony) [inline addons/bus/bus.py]: Please don't touch lines that are unrelated to your PR, to avoid noise in PRs and in history.

File: addons/hr/hr.py [hunk 2]
```diff
@@ -316,10 +316,11 @@ def onchange_department_id(self, cr, uid, ids, department_id, context=None):
         return {'value': value}
 
     def onchange_user(self, cr, uid, ids, user_id, context=None):
-        work_email = False
         if user_id:
-            work_email = self.pool.get('res.users').browse(cr, uid, user_id, context=context).email
-        return {'value': {'work_email': work_email}}
+            user_obj = self.pool.get('res.users').browse(cr, uid, user_id, context=context)
```
Comment (odony) [inline addons/hr/hr.py]: Looks ok, but this is a new feature, so it should target `master`, not 8.0 (in order to comply with our [stable policy](https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean)).
We can cherry-pick it in master in order to save you the extra PR, if you like.

Some (minor) extra nitpicking:
 - `user_obj` is a user browse_record, so it should be named `user`
 - since you're touching those lines already, you can simplify `self.pool.get('res.users')` into `self.pool['res.users']`
 - the dict returned with the data would be much better split into multiple indented lines
 - the dict returned when there is no `user_id` can simply be `{}`, the empty `value` is unnecessary

Thanks for the contribution!

File: addons/bus/bus.py (hunk 1 above)
Comment (Briksins) [inline addons/bus/bus.py]: Uh sorry, PyCharm was complain on semicolon, and that change was too small for separate commit, but I will not do like that again, sorry!

File: addons/hr/hr.py (hunk 2 above)
Comment (Briksins) [inline addons/hr/hr.py]: Yes please, cherry-pick it for me if you can.
I'm new to Odoo, need to figure out what is what's and where have to be committed, but I will do my best to help you out in the future. I will remember all advises for the next commit.

Comment (tde-banana-odoo) [inline addons/hr/hr.py]: I would also write user.image on employee.image, instead of using image_medium. Other fields are automatically computed.

Comment (Briksins) [inline addons/hr/hr.py]: I agree about image, as i understand from debugging there are 3 type of images - native, medium and small, however Employee had place only for medium image, so I wasn't sure if I can assign native image! didn't want to break anything

**Comment by Briksins** [comment]
Why legal/cla check failed? I did add required file (syncordia.md) signed by our director :(

File: doc/cla/corporate/syncordia.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+declaration.
+
+Signed,
+
+Vadim Zenin vadim.zenin@syncordiahealth.ie https://github.com/Vadim-Zenin
+
+List of contributors:
+
+Vadim Zenin vadim.zenin@syncordiahealth.ie https://github.com/Vadim-Zenin
+Vadims Briksins vadim.bryksin@syncordiahealth.ie https://github.com/Briksins
```
Comment (nim-odoo) [inline doc/cla/corporate/syncordia.md]: You commit with Briksins@gmail.com, while you sign with vadim.bryksin@syncordiahealth.ie
The email address should be the same ;-)

Comment (Briksins) [inline doc/cla/corporate/syncordia.md]: oh that's work email, however github on my own account and I used my own email... damn so many problems with my first commit...

Comment (nim-odoo) [inline doc/cla/corporate/syncordia.md]: What should I do then? Use your personal address or your work address in the CLA?
Thanks!

Comment (Briksins) [inline doc/cla/corporate/syncordia.md]: no, I think it should be work email, as im contributing from the company name, not as individual

maybe you would cancel this pull request and I'll recommit all again with all suggestion improvements into correct branch under correct mail?

Comment (nim-odoo) [inline doc/cla/corporate/syncordia.md]: @Briksins : yes, that would be better.
Thanks!

**Comment by Briksins** [comment]
Closing pull request and recommit to master branch with mentioned changes

File: doc/cla/corporate/syncordia.md (hunk 3 above)
Comment (Briksins) [inline doc/cla/corporate/syncordia.md]: 1) "Image" didn't work out, it must be 'image_medium'
2) Master to where I am suppose to commit the change is Odoo V9, but I was doing fix specifically for Odoo V8 so it seems I did right to commit to branch 8.0. anyway doing everything the way you asked

Comment (odony) [inline doc/cla/corporate/syncordia.md]: @Briksins The thing is, it's not a fix, it's an improvement/feature. Hence the target for Odoo 9, as a general policy. If we start accepting everybody's wishlist in stable series we drastically increase the risk of regressions and side-effects, in a series that is in production in thousands of companies. Many companies periodically update to the latest stable revision without specifically re-testing everything (that's what *stable* is about), so the cost of any side-effect is very high. The "improvements" are simply not worth the risk in general, so our [policy](https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean) is quite clear about that. When included in the next major version, these improvements will have a lot more battle-testing and will only be put in production after a proper upgrade/staging process. Thanks again for your contributions and your quick answers.
